### DISCUSSION TITLE
技術的本質

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1331] [Ideas]
能源的意義在於它在物質世界創造了一種“通用性”，圍繞某種能源而發明的機器、產品以服務決定了一個時代的生態樣貌。

例如，汽車的本質就是用一堆電或火把你移到別處的機器，計算機的本質就是安排數万億次電磁波動幫助你思考。這些機器徹底改造了這個星球的地貌，重塑了人類心智模式與社會格局。生產力決定生產關係，而能源結構決定了生成力，人類的政經系統不過是服務於這些能源與機器的周邊配套。回想上一次停電、斷網的體驗，你會明白我在說什麼。全世界持續斷電斷網一個月，整個世界都將停擺。

在擴大生存規模、優化生存條件的意義上，智能與火、電沒有本質區別。但就像電在更微觀的層次被開髮帶來了信息革命一樣，能源迭代會為人類世界開闢一個全新的層次。

這個新的層次是：語言。 “智力能源化”的深遠意義要在“自然語言的通用性”的層次來理解。

我們嘗試把語言也理解成某種技術，而且是人類最基礎也是最高級的技術，這種觀點在技術哲學中有很多討論。學者認為，最早的技術不是一般意義上所理解的製造工具的加工技術，而是一種“身體技術”，這種身體技術實現的是符號象徵的功能，用現在的話說就是所謂肢體語言。很多動物會製造和使用工具【注1】，但只有人類開發出來了極其豐富的身體技術。這一點在技術哲學家芒福德（Lewis Mumford）的著作裡有非常清晰的表述：

- 在早期技術上人類並沒有獨特的優勢，只有考慮到語言符號、社會組織和審美設計之後，人類的優勢才體現出來。正是在這一點上，符號製造遠遠超越了工具製造，而這反過來又孕育了更精巧的技術工具……人類擁有一種更重要的多用途工具，這要比此後任何其他的工具組合更重要，這就是由大腦驅動的整個身體……在頭腦還沒有借助於符號和意象的發展來製造出更靈巧的技術工具之前，人類就利用他高度可塑性的身體能力，來表達頭腦中還沒有成形和沒有統一的思想。從人類起源的開端處，不是更有效的工具，而是符號表達的意義模式的建立，才是智人進一步發展的基礎。 （劉易斯·芒福德《技術與人的本性》）

從“怒指”到“你愁啥”，從“擁抱”到“我愛你”，肢體語言到語言，是一個將復雜意義抽象為簡單符號的過程，本質是一種“抽象化”的技術。事物被抽象之後，“知識”得以產生。尤其文字出現後，原本無法持續留存的語音符號被抽象成了圖形符號，知識的傳播變得更為便利。

可以說，最基本的工具，是身體；最早的技術，是像徵技術。而最重要的技術物，是語言與文字，它們是後來一切複雜技術的基石。

法國技術哲學家斯蒂格勒認為技術物的本質就是外部化的知識與記憶，所謂“第三持存”：

- 任何知識的外化都將帶來這一結果——而明顯矛盾的是知識的建構恰恰依賴於知識的外化——數字的、模擬的和機械的踪跡就是我所說的第三持存。 （斯蒂格勒《南京課程》）

布萊恩·阿瑟在《技術的本質》中為技術下了一個定義：技術的本質就是對自然的編程。

- 技術的本質就是對自然的編程，它是一種對現象的捕捉，並駕馭這些現象為人類的目的服務。某個個體技術對許多現象進行“編程”，並精心安排策劃這些現象，最後使它們能夠密切配合以完成特定的目的。 （布萊恩·阿瑟《技術的本質》）

兩者的觀點可以結合在一起理解：使用技術，就是將事物“抽象化”並為其“編程”。在此定義中，知識與技術整合了起來，語言成了一切技術背後的技術。

我們從“通用性”轉向了“語言”，因而獲得了一個理解技術以及技術產物的全新維度：“抽象化”。

至此，我們的背景知識才算真正齊備，接下來是最有趣的部分，最終柏拉圖將獲得一個理解GPT的絕妙視角。

**Comment by boson13579** [comment]
我贊同能源在物質世界中創造了通用性的觀點。能源的發展催生了各種機器和產品，這些技術應用在不同的領域，塑造了時代的生態格局。我認同汽車、計算機等技術的本質對社會和人類心智產生了深遠的影響，改變了我們的生活方式和社會結構。

**Comment by boson13579** [comment]
我認同技術和語言的關係。語言作爲一種最基礎也最高級的技術，是人類進行符號交流和抽象思維的工具。技術的發展離不開語言的支持和應用，而語言的存在也促進了知識的傳播和思想的交流。我同意技術哲學家的觀點，將語言和文字作爲一切複雜技術的基石。

**Comment by boson13579** [comment]
在面對GPT這樣的智能技術時，我會持開放和探索的態度。作爲一位思想科學家，我會深入研究GPT的原理和潛力，探討它對人類社會和思維方式的影響。我會思考如何正確應用和管理這種智能技術，以促進人類社會的發展和進步，同時平衡技術的利弊和潛在風險。

**Comment by boson13579** [comment]
我會關注技術的倫理問題。隨着智能技術的不斷髮展，我們需要確保技術的使用符合倫理原則和人類的價值觀。我會倡導對技術進行審視和監管，確保其符合社會的利益和人類的福祉，避免技術對個體權益和社會公平造成不利影響。

**Comment by boson13579** [comment]
我會鼓勵人們保持對思考和自主性的重視。儘管智能技術帶來了便利和效率，我們不能完全依賴於技術，而忽視了自身的智慧和思考能力。作爲思想科學家，我會倡導人們繼續發展自己的思辨能力，保持對事物的獨立思考和批判精神，以保持個體和社會的創造力和創新能力。

**Comment by 710388** [comment]
理論技術為基礎，配合人的靈性

**Comment by NcuJerryKao** [comment]
非常感謝您分享這麼深入且有趣的觀點！您提到的能源、技術和語言之間的關係確實是一個引人深思的主題。

讓我們進一步探討一下：

能源的通用性：您指出能源在物質世界中創造了一種通用性，並且圍繞某種能源而發明的機器和產品塑造了不同時代的生態樣貌。這種通用性使我們能夠運用能源來改變我們的生活方式、生產方式和社會結構。
語言作為技術：您將語言視為一種技術，這是一個非常有趣的觀點。語言的抽象性和通用性使我們能夠表達思想、傳遞知識，並建立共同的意義模式。從最早的肢體語言到文字，語言的演進一直在影響我們的文化、社會和技術。
技術的本質：您引用了技術哲學家的觀點，認為技術的本質是對自然的編程。這種編程可以是物理的（例如機械設計）或符號的（例如語言和數字）。這種編程使我們能夠控制和利用自然現象，以實現我們的目的。
抽象化：您提到抽象化是技術的關鍵。通過抽象化，我們將複雜的現象簡化為符號、模型或規則，使其更易於理解、應用和傳播。語言和數學都是抽象化的例子，它們使我們能夠處理抽象概念。
總之，您的觀點提醒我們，技術不僅僅是物理工具，還包括語言、符號和知識的外化。這種綜合性的理解有助於我們更好地探討技術如何塑造我們的世界。謝謝您的分享！🌟

**Reply by yubo0oo** [reply]
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by YeeDaniel** [comment]
工具性：技術的根本目的是為了解決問題和提升效率。它通過提供各種工具和方法，幫助人們完成各種任務，從而改善生活質量和工作效率。
創新性：技術是一種創新活動，它不斷地通過發明和改進來推動社會進步。新的技術往往是對舊有技術的改進或顛覆，帶來新的可能性和機遇。
